background/background.js

```diff
@@ -0,0 +1,34 @@
+// This file is loaded as ESM.
+
+// Transition from Scratch Messaging Extension
+import "./transition.js";
+
+// Load idb
+// TODO before merging: can we run this as an ES module?
```

Comment:
```suggestion
// Running as ESM is possible but requires refactor; individual JS file should import
```

Reply:
Hmm, I meant running in strict mode. I don't intend to use `import {} from ""` syntax.

Reply:
Let me clarify. My code comment here means "can we do this (the import on the line below)"?

The idb.js file was never used with type=module, so I'm not 100% sure if it will behave properly by importing it this way, which enables strict mode.